albumentations/core/bbox_utils.py

```diff
@@ -62,11 +62,15 @@ def __init__(
         label_fields: Optional[Sequence[str]] = None,
         min_area: float = 0.0,
         min_visibility: float = 0.0,
+        min_width: float = 0.0,
+        min_height: float = 0.0,
         check_each_transform: bool = True,
     ):
         super(BboxParams, self).__init__(format, label_fields)
         self.min_area = min_area
         self.min_visibility = min_visibility
+        self.min_width = min_width
+        self.min_height = min_height
```

Comment:
Please, add new args to `_to_dict`

Reply:
I had not noticed `_to_dict`. Thanks.

da83ed0